File: PA2_Client/src/edu/ucdenver/tournament/Player.java
package edu.ucdenver.tournament;

public class Player {
    private String name;
    private int age;
    private double height;
    private double weight;

    public Player(String name, int age, double height, double weight) {
        this.name = name;
        this.age = age;
        this.height = height;
        this.weight = weight;
    }

    public String getName() {
        return this.name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getAge() {
        return this.age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public double getHeight() {
        return this.height;
    }

    public void setHeight(double height) {
        this.height = height;
    }

    public double getWeight() {
        return this.weight;
    }

    public void setWeight(double weight) {
        this.weight = weight;
    }

    @Override
    public String toString(){
        return String.format("\nPlayer Name: %s \nPlayer Age: %d \nPlayer Height: %f \nPlayer Weight: %f \n",
                this.name, this.age, this.height, this.weight);
    }
}

File: PA2_Server/src/edu/ucdenver/Tournament/Player.py


class Player:
    """keeps track of the _name, _age, _height, and _weight of a player"""

    def __init__(self, name: str, age: int, height: float, weight: float):
        """initializing player object"""
        self._name = name
        self._age = age
        self._height = height
        self._weight = weight

    @property
    def name(self):
        """returning the name of the player"""
        return self._name

    @property
    def age(self):
        """returning the age of the player"""
        return self._age

    @property
    def height(self):
        """returning the height of the player"""
        return self._height

    @property
    def weight(self):
        """returning the weight of the player"""
        return self._weight

    def __str__(self):
        """representing player object as a string"""
        string = 'Name: {:s} || Age: {:s} || Height: {:s} || Weight: {:s}'.format(self._name, self._age, self._height,
                                                                     self._weight)
        return string
